cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,6 +25,8 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
 
 def events_check(updates=False):
     form_bug = BZ(1485953, forced_streams=['5.7', '5.8', 'upstream'])
```

Comment:
This line can be removed now? If not completely then 5.7 and 5.8

Reply:
@sshveta 
This bug is closed as ```WONTFIX```. Hence removing only ```5.7 and 5.8``` dependency.